cluster/kubernetes/resource/load.go

```diff
@@ -14,8 +14,11 @@ import (
 
 // Load takes paths to directories or files, and creates an object set
 // based on the file(s) therein. Resources are named according to the
-// file content, rather than the file name of directory structure.
-func Load(base string, paths []string) (map[string]resource.Resource, error) {
+// file content, rather than the file name of directory structure. The
+// `fallbackNamespace` is assigned to any resource that doesn't
+// specify a namespace; it's only used for identification, and not put
+// in the definition.
+func Load(base, fallbackNamespace string, paths []string) (map[string]resource.Resource, error) {
```

Comment:
In order to consolidate this with #1668 I was thinking we could create something like, 

```
type ParsingContext interface {
  DefaultNamespace() string
  // in #1692 
  // ResourceScope (groupVersion string, string kind) int
}
```

Reply:
That was my thought on combining these as well. If knowing the default is not necessary for #1668, this can be adapted to add it later.